.github/workflows/elixir.yml

```diff
@@ -17,8 +17,8 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: '1.8'
-              otp: '21.3'
+              elixir: '1.12'
```

Comment:
The general strategy is to list the oldest Elixir version we support along with the oldest Erlang version that works with it. In this case, I believe that means the first pair should be Elixir 1.10 / OTP 21.3, and then the next pair would be Elixir 1.12 / OTP 24.0

Reply:
Working on the changes, thank you for the clarification on that